easybuild/easyconfigs/l/lftp/lftp-4.9.2-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'ConfigureMake'
+
+name = 'lftp'
+version = '4.9.2'
+
+homepage = 'http://lftp.yar.ru'
+description = """LFTP is a sophisticated ftp/http client, and a file transfer program supporting
+  a number of network protocols. Like BASH, it has job control and uses the readline library for
+  input. It has bookmarks, a built-in mirror command, and can transfer several files in parallel.
+  It was designed with reliability in mind."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://lftp.yar.ru/ftp/']
+sources = [SOURCE_TAR_GZ]
+# checksums = ['19f3a4236558fbdb88eec01bc9d693c51b122d23487b6bedad4cc67ae6825fc2']
```

Comment:
@levlafayette could you say why you've commented out the `checksum` here please? Or was that maybe accidental?

Reply:
... Can't remember! In any case it's wrong, the checksum should be

a37589c61914073f53c5da0e68bd233b41802509d758a022000e1ae2076da733